frontend/src/views/HistoryView.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useEffect, useState } from 'react'
+import HisotryCard from '../components/HistoryCard'
+import ProgressBar from 'react-bootstrap/ProgressBar';
+import Dropdown from 'react-bootstrap/Dropdown';
+
+
+
+//xport default ScreenreaderLabelExample;
+
+export default function History() {
+    const now = 60;
+    return (
+        <><h1>Calendar View</h1>
+
+            <Dropdown>
+                <Dropdown.Toggle variant="success" id="dropdown-basic">
+                    Month
+                </Dropdown.Toggle>
+
+                <Dropdown.Menu>
+                    <Dropdown.Item href="#/action-1">Jan 2023</Dropdown.Item>
+                    <Dropdown.Item href="#/action-2">Feb 2023</Dropdown.Item>
+                    <Dropdown.Item href="#/action-3">Mar 2023</Dropdown.Item>
+                    <Dropdown.Item href="#/action-3">Apr 2023</Dropdown.Item>
+                </Dropdown.Menu>
+            </Dropdown>
+            <div>
+            <HisotryCard _id={'111'} name={'water plants'} date={'2023-02-21'} description={'water three plants'} image={'/'}></HisotryCard>
```

Comment:
Maybe save these hardcoded examples somewhere, to make sure that incorporating the backend API calls won't be as complex. You can refer to the "TodoList.tsx" file for an example that uses a loop to draw each card, and use a similar loop here. Also, "HisotryCard" is spelled wrong, but minor fix!